src/reducers/session.ts

```diff
@@ -68,9 +69,12 @@ export default function session(
         }),
       };
     }
+    case SESSION_GET_SERVERINFO: {
```

Comment:
How is this similar / different to `SESSION_BUCKETS_REQUEST` and `SESSION_BUSY`?

Maybe what was missing here was that we forgot to call `sessionBusy(<bool>)` in the `getServerInfo` saga?

Reply:
Hmm, that is a bit cleaner. Switching to that.